lib/stripe/util.rb

```diff
@@ -68,19 +68,29 @@ def self.object_classes
     #
     # ==== Attributes
     #
-    # * +resp+ - Hash of fields and values to be converted into a StripeObject.
+    # * +data+ - Hash of fields and values to be converted into a StripeObject.
     # * +opts+ - Options for +StripeObject+ like an API key that will be reused
     #   on subsequent API calls.
-    def self.convert_to_stripe_object(resp, opts)
-      case resp
+    # * +response+ - An object containing information about the API response
+    #   that produced the data which is hydrating the StripeObject.
+    def self.convert_to_stripe_object(data, opts, other_opts = {})
```

Comment:
Ugh, this should be an optional argument (just have `response` instead of `other_opts`), but it breaks the 1.9 build. I wonder if we should just rip off that band-aid.